var/spack/repos/builtin/packages/stacktrace/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Stacktrace(CMakePackage):
+    homepage = "https://github.com/AdvancedMultiPhysics/StackTrace"
+    git = "https://github.com/AdvancedMultiPhysics/StackTrace.git"
+
+    maintainers("bobby-philip", "gllongo", "rbberger")
+
```

Comment:
Would you be able to add the `license` directive between maintainers and versions?